### PR TITLE
Internal: Improve the Nested Tabs aria-labels [ED-12065]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 1]
```diff
@@ -1082,6 +1082,8 @@ protected function render_tab_titles_html( $item_settings ): string {
 			$css_classes[] = 'elementor-animation-' . $item_settings['settings']['hover_animation'];
 		}
 
+		$aria_label = esc_html__( '. Open links with Enter or Space, close with Escape, navigate with Arrow Keys', 'elementor' );
```
Comment (rami-elementor) [inline modules/nested-tabs/widgets/nested-tabs.php:1085]: You should not start a translation string with a period. It's had to translate this type of string.

Developers, that see the code, understand the logic. But translators don't.

**Review by rami-elementor — CHANGES_REQUESTED** [review]

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 2]
```diff
@@ -1245,6 +1247,8 @@ protected function content_template() {
 					tabId = item.element_id;
 				}
 
+				const ariaLabel = '<?php echo esc_html__( '. Open links with Enter or Space, close with Escape, navigate with Arrow Keys', 'elementor' ); ?>';
```
Comment (rami-elementor) [inline modules/nested-tabs/widgets/nested-tabs.php:1250]: Coding Standards: You are using camel case in PHP...

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 1 above)
Comment (hein-obox) [reply modules/nested-tabs/widgets/nested-tabs.php:1085]: Updated.

**Review by davseve — DISMISSED** [review]

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 3]
```diff
@@ -1253,7 +1253,7 @@ protected function content_template() {
 					'aria-selected': 1 === tabCount ? 'true' : 'false',
 					'tabindex': 1 === tabCount ? '0' : '-1',
 					'aria-controls': 'e-n-tab-content-' + tabUid,
-					'aria-label': '<?php echo esc_html__( 'Open tab (Enter or Space) | Return (Escape)', 'elementor' ); ?>',
```
Comment (rami-elementor) [inline modules/nested-tabs/widgets/nested-tabs.php:1256]: This `aria-label` is not required as it duplicates the title from the inner text. You can remove the entire attribute.

**Review by rami-elementor — CHANGES_REQUESTED** [review]

**Review by rami-elementor — APPROVED** [review]

**Pull request merged**